Emby.Server.Implementations/Updates/InstallationManager.cs

```diff
@@ -441,7 +336,7 @@ private async Task PerformPackageInstallation(PackageVersionInfo package, Cancel
             // Always override the passed-in target (which is a file) and figure it out again
             string targetDir = Path.Combine(_appPaths.PluginsPath, package.name);
 
-// CA5351: Do Not Use Broken Cryptographic Algorithms
+            // CA5351: Do Not Use Broken Cryptographic Algorithms
 #pragma warning disable CA5351
```

Comment:
Is this really the proper formatting for compiler flags? It seems odd to me.

Reply:
Ref: https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/preprocessor-directives/preprocessor-if#examples